src/qt/pivx/topbar.cpp

```diff
@@ -541,23 +540,50 @@ void TopBar::updateBalances(const CAmount& balance, const CAmount& unconfirmedBa
     CAmount pivAvailableBalance = balance;
     // zPIV Balance
     CAmount matureZerocoinBalance = zerocoinBalance - unconfirmedZerocoinBalance - immatureZerocoinBalance;
+    bool fHaveZerocoins = zerocoinBalance > 0;
```

Comment:
Would move this below the comment: "// Update display state and/or values for zPIV balances as necessary". It would make the code easier to follow.

Reply:
done